hack/print-workspace-status.sh

```diff
@@ -35,11 +35,17 @@ if [ ! -z "$(git status --porcelain)" ]; then
     GIT_STATE="dirty"
 fi
 
+KIND_STATE=""
+if docker version > /dev/null 2>&1; then
+    KIND_STATE=$(docker ps -f 'label=io.k8s.sigs.kind.cluster' --format='{{.ID}}' | tr '\n' '_')
+fi
```

Comment:
This solution for invalidating the bazel cache isn't *too* awful. The only drawback, is:

* If a cluster is *not* running and a kind_cluster target is invoked, all stamped targets will be rebuilt
* This will cause a cluster to be created, thus causing the KIND_STATE var to change
* If the same target is run again, *all* stamped targets will be rebuilt again. The kind_cluster targets can handle this via the `PERSISTENT_KUBECONFIG` var, but it *does* result in some extra GoLink and container_image steps being run.